src/main/java/pl/simpleascoding/tutoringplatform/domain/advertisement/Advertisement.java

```diff
@@ -0,0 +1,42 @@
+package pl.simpleascoding.tutoringplatform.domain.advertisement;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import pl.simpleascoding.tutoringplatform.domain.user.User;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+
+@Data
+@Entity
+@Table(name = "advertisement")
+@NoArgsConstructor
+public class Advertisement {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @NotNull
+    private AdvertisementCategory category;
```

Comment:
Please add on this field proper annotation with proper attribute: 
```java
@Enumerated(...)
```
**Information related to:**
https://stackabuse.com/guide-to-jpa-with-hibernate-basic-mapping/
https://bykowski.pl/hibernate-6-najprzydatniejsze-adnotacje-przy-modelowaniu-klas-encji/
https://doraprojects.net/questions/5450902/hibernate-enumerated-mapping